docs/applications/visualisation.py

```diff
@@ -127,4 +127,20 @@ def show_dislocation(system, scale=0.5, diamond_structure=False, add_bonds=False
                     });
                  """
     view._js(tooltip_js)
+    return view
+
+
+def interactive_view(system, scale=0.5):
```

Comment:
I moved this function to `show_dislocation()` that automatically colours the atoms to according to CNA analysis with ovito. Did you try to use it? This also allows to show dislocations as arrows. If it works for your systems it would be nice to use that to be consistent.